docs/source/conf.py

```diff
@@ -108,7 +108,7 @@
 # The default options for autodoc directives.
 # https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#confval-autodoc_default_options
 autodoc_default_options: Any = {
-    # If set, autodoc will generate document for the members of the target module, class or exception.  # noqa: E501
+    # If set, autodoc will generate document for the members of the target module, class or exception.  # noqa: E501, pylint: disable=line-too-long
```

Comment:
Maybe just split the comment on 2 lines?

Reply:
now that I have the max-line-length passed on the command line this is OK. Removed the pylint ignore instruction, too...